examples/open_loop_control_urdfbot/generate.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# generate series of actions for fetch agent on different parts such as moving, arm and gripper.
+
+import numpy as np
+import pandas as pd
+
+wheel_joints = ["wheel_joint_r", "wheel_joint_l"]
```

Comment:
- `["wheel_joint_l", "wheel_joint_r"]` (left before right makes more sense, and the order shouldn't matter because it gets passed in as a named dictionary anyway)
- `["gripper_finger_joint_l", "gripper_finger_joint_r"]`

Reply:
Actually I don't think these arrays of names are necessary.

Reply:
fixed